File: FertiWeb/src/main/java/dto/DTOFuenteXRecomendacion.java
package dto;

import dominio.Unidad;
import lombok.Data;

@Data
public class DTOFuenteXRecomendacion {
	
	private Long codigoFuenteXRecomendacion;
	private DTOFuente fuente;
	private Double cantidad;
	private Unidad unidad;
	private Long codigoRecomendacion;

}

File: FertiMobile/src/app/dto/dto-fuente-x-recomendacion.ts
import { DTOFuente } from './dto-fuente';
import { Unidad } from 'src/app/dominio/unidad';
export class DTOFuenteXRecomendacion {
    codigoFuenteXRecomendacion: number;
    fuente: DTOFuente;
    cantidad: number;
    unidad: Unidad;
    codigoRecomendacion: number;
}
